config/fabric_module_set_admin_status.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+#
+import re
+import subprocess
+import time
+from swsscommon.swsscommon import SonicV2Connector
+from sonic_py_common.logger import Logger
+import utilities_common.cli as clicommon
+
+# Name: fabric_module_set_admin_status.py, version: 1.0
+# Syntax: fabric_module_set_admin_status <module_name> <up/down>
+def fabric_module_set_admin_status(module, state):
+    logger = Logger("fabric_module_set_admin_status.py")
+    logger.set_min_log_priority_info()
+
+    if not module.startswith("FABRIC-CARD"):
+        logger.log_warning("Failed to set {} state. Admin state can only be set on Fabric module.".format(module))
```

Comment:
This is wrong.  This is only applicable to Nokia platform.  May not for other vendor